plugins/cros-ec/fu-cros-ec-usb-device.c

```diff
@@ -698,25 +710,15 @@ fu_cros_ec_usb_device_jump_to_rw(FuCrosEcUsbDevice *self)
 						   &response,
 						   &response_size,
 						   FALSE,
-						   NULL)) {
+						   &error_local)) {
 		/* bail out early here if subcommand failed, which is normal */
+		g_debug("ignoring failure: jump to rw: %s", error_local->message);
 		return TRUE;
 	}
 
 	/* Jump to rw may not work, so if we've reached here, initiate a
 	 * full reset using immediate reset */
-	subcommand = FU_CROS_EC_UPDATE_EXTRA_CMD_IMMEDIATE_RESET;
-	fu_cros_ec_usb_device_send_subcommand(self,
-					      subcommand,
-					      command_body,
-					      command_body_size,
-					      &response,
-					      &response_size,
-					      FALSE,
-					      NULL);
-
-	/* success */
-	return TRUE;
+	return fu_cros_ec_usb_device_reset_to_ro(self, &error_local);
```

Comment:
error, no?

Reply:
hmm.. no, fu_cros_ec_usb_device_jump_to_rw() has no *error, and btw fu_cros_ec_usb_device_reset_to_ro() always return TRUE.